client/src/components/AddNewProfile.js

```diff
@@ -242,11 +247,11 @@ const AddNewProfile = () => {
         <Dropdown
           id="nationality"
           name="nationality"
-          value={profileData.nationalities}
+          value={profileData.nationality}
           fluid
           selection
           options={nationalityOptions}
-          onChange={handleDropdownAndDateChange}
+          onChange={nationalityHandler}
```

Comment:
I think we don't need to create a new function for this. I believe we can still use `handleDropdownAndDateChange` like before, and therefore delete the `nationalityHandler` function